doc/api.rst

```diff
@@ -403,6 +403,56 @@ Computation
 :py:attr:`~core.groupby.DataArrayGroupBy.where`
 :py:attr:`~core.groupby.DataArrayGroupBy.quantile`
 
+
+String manipulation
+-------------------
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor_method.rst
+
+   DataArray.str.capitalize
```

Comment:
can we do this for the `.dt` accessor too?

Reply:
sure, that's what I was planning to (and plotting, too, if that makes sense), and to add those to `Dataset`, too